packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json

```diff
@@ -751,23 +751,23 @@
   "hosting_tab_FTP_multiftp_table_ssh_state_NONE": "Désactivé",
   "hosting_tab_FTP_multiftp_table_state_rw": "Activé",
   "hosting_tab_FTP_multiftp_table_state_off": "Désactivé",
-  "hosting_tab_FTP_multiftp_table_ssh_state_active": "Activé",
-  "hosting_tab_FTP_multiftp_table_ssh_state_sftponly": "SFTP",
-  "hosting_tab_FTP_multiftp_table_ssh_state_none": "Désactivé",
+  "hosting_tab_FTP_multiftp_table_ssh_state_active": "FTP, SFTP and SSH",
```

Comment:
**question:** Why using word `and` into a `Messages_fr_FR.json` file?

```suggestion
  "hosting_tab_FTP_multiftp_table_ssh_state_active": "FTP, SFTP et SSH",
```

Reply:
Hell, thanks for the review. indeed, it doesnt make sense